templates/rancher/overlay/opt/corral/rancher/install-rancher.sh

```diff
@@ -7,15 +7,16 @@ helm repo update
 CORRAL_rancher_host=${CORRAL_rancher_host:="${CORRAL_fqdn}"}
 CORRAL_rancher_version=${CORRAL_rancher_version:=$(helm search repo rancher-latest/rancher -o json | jq -r .[0].version)}
 
-helm upgrade \
-  --install \
-  --create-namespace \
-  --set hostname="$CORRAL_rancher_host" \
-  --version "$CORRAL_rancher_version" \
-  --devel \
-  --wait \
-  -n cattle-system \
-  rancher rancher-latest/rancher
+args=("--install" "--create-namespace")
+
+if [ -z "${CORRAL_registry_fqdn}" ]; then
+  args+=("--set hostname=${CORRAL_rancher_host}" "--version ${CORRAL_rancher_version}")
+else
+  args+=("--set hostname=${CORRAL_rancher_host}" "--set extraEnv[0].name=CATTLE_SYSTEM_DEFAULT_REGISTRY" "--set extraEnv[0].value=${CORRAL_registry_fqdn}" "--version ${CORRAL_rancher_version}")
+fi
+
+args+=("--devel" "--wait" "-n cattle-system" "rancher rancher-latest/rancher")
```

Comment:
the --devel option shouldn't be necessary?

Reply:
Can you direct this to Izaac's PR? I am leveraging his work, and I am unsure of this :grin:

Reply:
The original package had it so I decided to keep it as-is